bazel/grpc_deps.bzl

```diff
@@ -181,7 +181,7 @@ def grpc_deps():
     )
 
     native.bind(
-        name = "libuv",
+        name = "uv",
```

Comment:
Is this change necessary?

Reply:
This change is grafted from the timer PR patch, it looks like the original "libuv" naming cannot be found by upstream bazel targets.